examples/extract_txt_with_page.rs

```diff
@@ -0,0 +1,98 @@
+use std::{cell::RefCell, collections::HashMap, env, fmt, rc::Rc};
+
+use lopdf::Document;
+use pdf_extract::{ConvertToFmt, OutputDev, OutputError, PlainTextOutput};
+
+fn main() {
+    let file = env::args().nth(1).unwrap();
+    let doc = Document::load(file).unwrap();
+    let mut output = PagePlainTextOutput::new();
+    pdf_extract::output_doc(&doc, &mut output).unwrap();
+
+    // print the text of each page
+    for (page_num, text) in output.pages {
+        println!("Page {}: {}", page_num, text);
+    }
+}
+
+struct PagePlainTextOutput {
+    inner: PlainTextOutput<OutputWarpper>,
+    pages: HashMap<u32, String>,
+    current_page: u32,
+    reader: Rc<RefCell<String>>,
+}
+
+struct OutputWarpper(Rc<RefCell<String>>);
```

Comment:
This should be OutputWrapper instead of OutputWarpper.

Reply:
done.